backend/api/models.py

```diff
@@ -1,19 +1,30 @@
 from django.db import models
 from django.contrib.auth.models import AbstractUser
 from django.utils.translation import gettext_lazy as _
+from django.core.validators import MinValueValidator
 from .managers import CustomUserManager
 
 
-    
 class CustomUser(AbstractUser):
     username = None
     email = models.EmailField(_("email address"), unique=True)
     first_name = models.CharField(_("first name"), max_length=150, blank=True, db_collation="case_insensitive")
     last_name = models.CharField(_("last name"), max_length=150, blank=True, db_collation="case_insensitive")
 
     USERNAME_FIELD = "email"
-    REQUIRED_FIELDS = ['first_name', 'last_name']   
+    REQUIRED_FIELDS = ['first_name', 'last_name']
     objects = CustomUserManager()
 
     def __str__(self):
         return self.email
+
+
+class Site(models.Model):
+    class Len_unit(models.TextChoices):
```

Comment:
Just curious, is there any particular reason to use a class instead of a list of values/tuple?

Reply:
I came across this [article](https://johnnymetz.com/posts/set-django-model-field-choices-in-frontend/) where they discuss sending choices to the frontend through an API. I'm interested in trying it out, but I encountered an issue. Swagger is not displaying the options method, and I'm having trouble adding it for testing purposes.